src/components/Sum.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+function Sum({ a, b }) {
+  return (
+    <p>
+      Sum of
+      {' '}
+      {a}
+      {' '}
+      and
+      {' '}
+      {b}
+      {' '}
+      is
+      {' '}
+      {Number(a) + Number(b)}
```

Comment:
you should use interpolation  here) 
[CODE STYLE] - Use string interpolation inside tag content(checklist)